examples/docs/go/.gitignore

```diff
@@ -0,0 +1,6 @@
+.shared
```

Comment:
The shared folder is something used by docsrv, but to be able to also build locally the documentations, we override this if not provided by docsrv with  `$(PWD)/.shared` in https://github.com/erizocosmico/ci-1/blob/c611664bab0ab64ac779c98df6d42b145f8ac6a2/examples/docs/cpp/Makefile#L14. That's why it's on the gitignore.